src/admin.tsx

```diff
@@ -139,29 +225,57 @@ export default function AdminPage() {
 
     // key = ambiance category, i.e., animalCrossing, bg3
     const elements = Object.entries(newData).map(([documentId, ambianceDisplay], index) => (
-      <TreeItem className='tree-item' key={documentId} nodeId={documentId} label={ambianceDisplay.friendlyName} >
+      <TreeItem
+        sx={{ padding: 1 }}
+        key={documentId}
+        nodeId={documentId}
+        label={<Typography variant='h6'>{ambianceDisplay.friendlyName}</Typography>}
+      >
+        <TreeItem
+          expandIcon={<TreeIcon tooltipText='Edit Category' icon={<BuildCircleIcon />} />}
+          collapseIcon={<TreeIcon tooltipText='Close Editor' icon={<BuildCircleIcon color='error' />} />}
+          key={documentId + 'edit_category'}
+          nodeId={documentId + 'edit_category'}
+          label={<p>Edit <i><b>{ambianceDisplay.friendlyName}</b></i> Category</p>}
+        >
+          <CategoryEditor editCategory={editCategory} deleteCategory={deleteCategory} categoryID={documentId} categoryDetails={data[documentId]}></CategoryEditor>
+        </TreeItem>
+        <Divider />
         {Object.entries(ambianceDisplay.videosBySubcategory).map(([subcategory, vids]) => (
-          <TreeItem key={documentId + subcategory} nodeId={documentId + subcategory} label={subcategory} >
+          <TreeItem
+            key={documentId + subcategory}
+            nodeId={documentId + subcategory}
+            label={<Typography variant='subtitle1'>{subcategory}</Typography>}
+          >
             {vids.map(vid => (
               <TreeItem
                 expandIcon={<TreeIcon tooltipText='Edit Video' icon={<BuildCircleIcon />} />}
-                collapseIcon={<TreeIcon tooltipText='Expand' icon={<BuildCircleIcon color='error' />} />}
-                key={vid.name}
-                nodeId={vid.name}
+                collapseIcon={<TreeIcon tooltipText='Close Editor' icon={<BuildCircleIcon color='error' />} />}
+                key={documentId + subcategory + vid.code + "edit_video"}
+                nodeId={documentId + subcategory + vid.code + "edit_video"}
```

Comment:
Updating keys and nodeIds throughout the tree to ensure that each one is truly unique